eclass/multibuild.eclass

```diff
@@ -1,23 +1,23 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 # @ECLASS: multibuild.eclass
 # @MAINTAINER:
 # Michał Górny <mgorny@gentoo.org>
 # @AUTHOR:
 # Author: Michał Górny <mgorny@gentoo.org>
-# @SUPPORTED_EAPIS: 4 5 6 7
+# @SUPPORTED_EAPIS: 4 5 6 7 8
 # @BLURB: A generic eclass for building multiple variants of packages.
 # @DESCRIPTION:
 # The multibuild eclass aims to provide a generic framework for building
 # multiple 'variants' of a package (e.g. multilib, Python
 # implementations).
 
 case "${EAPI:-0}" in
-	0|1|2|3)
+	[0-3])
 		die "Unsupported EAPI=${EAPI:-0} (too old) for ${ECLASS}"
 		;;
-	4|5|6|7)
+	[4-8])
```

Comment:
why not tighten this further?

Reply:
I'm waiting for the last EAPI=5 consumers to go, then I will be able to remove a lot of old code.

Reply:
but you're still declaring EAPI 4 valid?

Reply:
Yes because this commit is 'add new EAPI', 'remove old EAPIs' will come later.